pkg/handlers/app.go

```diff
@@ -184,6 +184,11 @@ func responseAppFromApp(a *apptypes.App) (*types.ResponseApp, error) {
 		return nil, errors.Wrap(err, "failed to check if rollback is supported")
 	}
 
+	targetKotsVersion, err := store.GetStore().GetTargetKotsVersionForVersion(a.ID, latestVersion.Sequence)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get target kots version")
```

Comment:
how about not failing for this?

Reply:
this will only fail if the app spec doesn't load for some reason, which will already be a problem for other fields. but i see your point. i can update